L1Trigger/TrackerTFP/plugins/ProducerAS.cc

```diff
@@ -0,0 +1,100 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+
+#include <string>
+
+using namespace std;
+using namespace edm;
+using namespace trackerDTC;
+
+namespace trackerTFP {
+
+  /*! \class  trackerTFP::ProducerAS
+   *  \brief  Associate fitted TTTracks with found TTTracks
```

Comment:
This comment is misleading. Perhaps change to "Associate TTTracks from end of TMTT chain with corresponding streamed tracks. i.e. Output and input of ProducerTT". And add that this is only required for analysis.

Reply:
the TMTT irritates me since we associate found TTTracks with fittest TTTracks for both the TMTT track finding and the tracklet track finding.